src/views/CharacterDetail/CharacterDetail.test.jsx

```diff
@@ -0,0 +1,34 @@
+import { screen, render, waitForElementToBeRemoved } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { MemoryRouter } from "react-router-dom";
+import App from '../../App';
+
+describe('<CharacterDetail />', () => {
+  it('renders selected character details', async () => {
+    render(
+        <MemoryRouter
+        initialEntries={["/character/16"]}
+        initialIndex={15}
```

Comment:
For our initialIndex here, we will want to have an initial index of 0, as we want to display the first string in the initialEntries array. You can actually omit this as if it is not present, it will default to zero.